pkg/testutil/fakes_test.go

```diff
@@ -390,6 +391,37 @@ func TestSimpleServiceLister(t *testing.T) {
 	assert.Equal(t, "test", service.Namespace)
 }
 
+func TestSimpleClusterRoleLister(t *testing.T) {
+	clusterConnections := GetManagedClusterConnections()
+	clusterConnection := clusterConnections["cluster1"]
+
+	cr := v1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test",
+		},
+	}
+	_, err := clusterConnection.KubeClient.RbacV1().ClusterRoles().Create(context.TODO(), &cr, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("can't create cluster role: %v", err)
+	}
+
+	l := simpleClusterRoleLister{
+		clusterConnection.KubeClient,
+	}
+	s := labels.Everything()
+	clusterRoles, err := l.List(s)
+	if err != nil {
+		t.Fatal(fmt.Sprintf("can't get cluster roles: %v", err))
+	}
+	assert.Equal(t, 1, len(clusterRoles))
```

Comment:
we should maybe get in the happen of putting the comment in there too - depends what kind of message we are getting from the testify/whatever if the test fails - if we can get enough information from that

Reply:
I think I've added comments to all the asserts.